packages/tpetra/core/test/FECrsMatrix/FECrsMatrix_UnitTests.cpp

```diff
@@ -145,8 +142,7 @@ class GraphPack {
   std::vector<std::vector<GO> > element2node;
 
   // NOTE: This is hardwired for 1D bar elements
-  typedef Kokkos::View<LO*[2], Kokkos::LayoutLeft, typename Node::device_type > k_element2node_type;
-  k_element2node_type k_element2node;
+  Kokkos::View<GO*[2], Kokkos::LayoutLeft, typename Node::device_type > k_element2node;
```

Comment:
Do we have a specific reason for using `LayoutLeft` here?  If not, let's drop it, to make the example shorter.

Reply:
OK.